docs/source/user_guide/calculation_plugins/pw.rst

```diff
@@ -34,8 +34,9 @@ Inputs
 * **pseudo**, class :py:class:`UpfData <aiida.orm.nodes.data.upf.UpfData>`
   One pseudopotential file per atomic species.
   
-  Alternatively, pseudo for every atomic species can be set with the **use_pseudos_from_family**
-  method, if a family of pseudopotentials has been installed.
+  Alternatively, pseudo for every atomic species can be set with the from the 
```

Comment:
Change this to:

If a pseudo potential family is uploaded, the :py:func:`~aiida.orm.nodes.data.upf.get_pseudos_from_structure` function can be used to automatically get the mapping of `UpfData` nodes for each kind in the `StructureData` one wants to use.
For example:
```
from aiida.orm.nodes.data.upf import get_pseudos_from_structure
pseudo_potenial_family = 'SSSP_efficiency'
structure = StructureData()  # Let's say this is a GaAs structure
pseudos = get_pseudos_from_structure(structure, pseudo_potenial_family)
```
The ``pseudos`` variable will now be a dictionary of the form:
```
pseudos = {
    'As': UpfData(),
    'Ga': UpfData(),
}
```
This can then be used directly in the process builder of for example a ``PwCalculation``.
```
builder = PwCalculation.get_builder()
builder.pseudos = pseudos
```